specs/citation_changes.proto

```diff
@@ -9,10 +10,9 @@ message CitationChanges {
 message CitationChange {
     string citing = 1;
     string cited = 2;
-    string doi = 3;
-    string pid = 4;
-    string url = 5;
-    string score = 6;
-    Status status = 7;
+    CitationChangeContentType content_type = 3;
```

Comment:
now it's ok, but in the future we should avoid reusing the ids (when a field is removed, its id  should be retired)